### PR TITLE
Remove deprecation for Zone and ZoneProps

### PR DESCRIPTION
**Pull request by NancyMcCallB**
Zone and ZoneProps are still required in UI 2.0 .

### CODE REVIEW

**Review by bsteinbk — APPROVED** [review]

File: common/api/summary/undeprecate-3.exports.csv [hunk 1]
```diff
@@ -0,0 +1,15 @@
+sep=;
+Release Tag;API Item
```
Comment (calebmshafer) [inline common/api/summary/undeprecate-3.exports.csv:2]: Looks like this was accidentally added?

Comment (NancyMcCallB) [reply common/api/summary/undeprecate-3.exports.csv:1]: I have no idea what that means. I assume the extract-api did that?

Comment (calebmshafer) [reply common/api/summary/undeprecate-3.exports.csv:1]: I was talking about the entire file. The extract-api summary files should only be per-package so not user where `undeprecate-3` comes from. I'd just delete the file.

Comment (NancyMcCallB) [reply common/api/summary/undeprecate-3.exports.csv:1]: Done.

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**